homeassistant/components/template/__init__.py

```diff
@@ -57,17 +57,29 @@ async def _reload_config(call: Event | ServiceCall) -> None:
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up a config entry."""
+    hass.data[DOMAIN] = {}
```

Comment:
Could this use `entry.runtime_data` instead? https://developers.home-assistant.io/blog/2024/04/30/store-runtime-data-inside-config-entry